REPOSITORIES.md

```diff
@@ -0,0 +1,78 @@
+# Repositories
+
+This document describes the lifecycle of repositories under the [falcosecurity](https://github.com/falcosecurity) along with their criteria, structure, scope, and status.
+
+## Criteria
+
+Repositories host specific parts of The Falco Project such as core codebase components, documentation, tools, libraries, or subprojects. The [falcosecurity](https://github.com/falcosecurity) GitHub organization owns repositories exclusively related to The Falco Project or its ecosystem.
+
+## Owners
+
+Repositories must contain [OWNERS](https://www.kubernetes.dev/docs/guide/owners/) files following the Kubernetes specific. OWNERS files are used to designate responsibility over different parts of the repository codebase and serve as the implementation mechanism for the two-phase code review process used by each project. 
```

Comment:
specific? I suspect you meant specification?

Reply:
Fixed. Thank you!